src/main/java/edu/greenblitz/robotName/shootingStateService/ShootingState.java

```diff
@@ -0,0 +1,64 @@
+package edu.greenblitz.robotName.shootingStateService;
+
+import edu.greenblitz.robotName.FieldConstants;
+import edu.greenblitz.robotName.subsystems.shooter.Pivot.Pivot;
+import edu.greenblitz.robotName.subsystems.shooter.Pivot.PivotConstants;
+import edu.greenblitz.robotName.subsystems.swerve.Chassis.SwerveChassis;
+import edu.greenblitz.robotName.utils.shootingCalculations.ShootingAngle;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
+
+import static edu.greenblitz.robotName.shootingStateService.ShootingPositionConstants.LEGAL_SHOOTING_ZONE;
+
+public class ShootingState {
```

Comment:
why is it called shooting state if it is not a state? maybe do
```suggestion
public class ShootingCalculations {
```
or something